samples/multiplatform/platform-js/build.gradle.kts

```diff
@@ -0,0 +1,65 @@
+import com.moowork.gradle.node.npm.NpmTask
+import com.moowork.gradle.node.task.NodeTask
+import org.jetbrains.kotlin.gradle.tasks.Kotlin2JsCompile
+
+plugins {
+    id("kotlin-platform-js")
+    id("com.moowork.node") version "1.2.0"
+}
+
+dependencies {
+    "expectedBy"(project(":platform-common"))
+    "implementation"(kotlin("stdlib-js"))
```

Comment:
why using `"xx"` instead of dirctly the method `implementation(kotlin(...)`?

Reply:
Intellij breaks time to time when it's defined without quotes